paper-server/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java

```diff
@@ -207,6 +209,11 @@
 
 @DelegateDeserialization(CraftOfflinePlayer.class)
 public class CraftPlayer extends CraftHumanEntity implements Player {
+    private static final PointersSupplier<Player> POINTERS_SUPPLIER = PointersSupplier.<Player>builder()
```

Comment:
Missing the display name here

Reply:
it has `.parent(CraftEntity.POINTERS_SUPPLIER)` which does contain the display name pointer

Reply:
That’s using the entity’s name _not_ displayname